### ISSUE TITLE
Unable to connect front-end to git branch when using GitLab connected repo

### ISSUE THREAD

**Issue by alexindrez** [aws-amplify/amplify-console#1532]
**Describe the bug**
When connecting the front-end Amplify deployment to a Git branch I am unable to proceed through the wizard to complete the procedure to deploy a front-end. When I get to step 2/2 there are no options available. The next button is disabled. I was able to edit the next button properties in Chrome dev tools and then click the button and it flows through the rest of the procedure as expected.

**To Reproduce**

Went in to Amplify console (https://console.aws.amazon.com/amplify/)
Clicked on app -> Front end environments
Clicked Connect Branch button
Step 1 select repository ( GitLab is connected ), waited for options to load
Selected branch from branch drop down
Selected back-end env from second drop down
Clicked Next
Step 2: no options available, next button is disabled

**Expected behavior**

The next button should be enabled if there are no other inputs available/required to complete the "wizard"

**Screenshots**
![Screen Shot 2021-02-02 at 12 57 34 PM](https://user-images.githubusercontent.com/4501530/106649089-1c897100-655f-11eb-8249-af4ff77e2682.png)

**Amplify CLI Version (if issue tied to Amplify CLI) **
 - CLI Version: 4.41.2
 - OS: Mac
 - Node Version. 12.16
 - Browser type: Google Chrome

**Additional context**
I tried this in Firefox and experience the sam behavior.

**Comment by ganipcanot** [comment]
Could you share your `appId` and `region`?  We previously had an issue with gitlab usernames which we fixed - possibly we missed something?  Would you mind sharing your gitlab username?  If you're not comfortable sharing it here you can send it to `aws-amplify-customer@amazon.com` please include the this github issue (https://github.com/aws-amplify/amplify-console/issues/1532) so we know it's you!

**Comment by allanitis** [comment]
Also experiencend this since beginning using Amplify with Gitlab since 2019. Once clicking "Connect branch", we just wait 10-15 seconds for the branches and environments to load. Then choose the branch and backend env, then click Next. 

If we click Next, it just stays disabled.

**Comment by alexindrez** [comment]
@ganipcanot Sorry I just noticed an email about this in my inbox when @allanitis posted a reply. I just sent an email with my projectId, region and Gitlab username to the address as requested. Thank you!

@allanitis I see this behavior no matter how long I wait before clicking the next button. The button state in the dom simply doesn't get updated, and I can edit it dev tools, hit submit and all goes through fine. I figured I'd post this in case other folks were running into the same issue.
